.github/workflows/go-integration.yml

```diff
@@ -48,11 +48,11 @@ jobs:
 
       - name: Start docker
         run: |
-          docker compose -f dev/docker-compose.yml up -d
+          docker compose -f recipe/docker-compose.yml up -d
           sleep 10
       - name: Provision Tables
         run: |
-          docker compose -f dev/docker-compose.yml exec -T spark-iceberg ipython ./provision.py
+          docker compose -f recipe/docker-compose.yml exec -T spark-iceberg ipython ./provision.py
```

Comment:
wouldn't this cause a conflict with the scanner tests that are starting the test containers locally?

Reply:
not really, should work totally fine.

Reply:
it's not going to run into an issue with the ports already being used?

Reply:
Not really, I left a check - if ports presented - do not start a container.
It's quite handy, since you can run it in dual mode both running on a side and per test run

Reply:
Ah, very cool!